server/cluster.go

```diff
@@ -897,3 +1061,44 @@ func (c *Cluster) invalidateRemoteSubs() {
 		s.presTermDirect(topicsToTerminate)
 	}
 }
+
+func (sess *Session) topicProxyWriteLoop() {
+	defer func() {
+		sess.closeRPC()
+		globals.sessionStore.Delete(sess)
+		sess.unsubAll()
+	}()
+
+	for {
+		select {
+		case msg, ok := <-sess.send:
+			if !ok || sess.clnode.endpoint == nil {
+				// channel closed
+				return
+			}
+			srvMsg := msg.(*ServerComMessage)
+			if srvMsg.sessOverrides == nil {
+				log.Println("cluster: proxy session overrides not specified - ", sess.sid)
```

Comment:
How can it happen? Only if there is a bug somewhere? If so we should panic instead.